docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -0,0 +1,70 @@
+# Archiveren (voor dummies)
+
+## Gerelateerde attributen
+
+* `Zaak.Archiefnominatie` Aanduiding of het zaakdossier blijvend bewaard of na een bepaalde termijn vernietigd moet worden:
+
+    * `null`
```

Comment:
Ik heb de structuur en de volgordelijkheid van het zetten van attributen niet goed in mijn hoofd, vandaar de vraag in welke situatie `Zaak.Archiefnomiatie` waarde `nulll` krijgt?

Reply:
`null` is de technische invulling van "optioneel". Het is dus een vaste enumeratie (vernietigen/bewaren) maar hoeft geen waarde te hebben.